### DISCUSSION TITLE
Add a --help cli argument to show currently supported cli arguments

### DISCUSSION THREAD

**Discussion by thiswillbeyourgithub** [rustdesk/rustdesk#9484] [General]
Hi,

As I explained in [this comment](https://github.com/rustdesk/rustdesk/discussions/3980#discussioncomment-10772928):
> For anyone still wondering what command line arguments are currently available, take a look at [the file src/core_main.rs](https://github.com/rustdesk/rustdesk/blob/master/src/core_main.rs) and do ctrl+f for '--'.
> 
> I'm not sure there are other arguments than those.

It would be great if you could add a simple '--help' or '-h' argument that would at least simply show the list of supported cli argument.

On a related note, I was planning on adding rustdesk to [tldr pages](https://github.com/tldr-pages/tldr) so that `tldr rustdesk` would show the most useful comands but I can't because there's  no --help yet. I think it would be great if you did it. [Here are their contributing guidelines](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md)

Edit: this was also requested here: https://github.com/rustdesk/rustdesk/discussions/3980#discussioncomment-9687526

**Comment by alx-xlx** [comment]
You can also do the search in the docs - https://github.com/rustdesk/doc.rustdesk.com
Just found one command `rustdesk --config $rustdesk_cfg` in there

the help cmd is very much needed, I can't figure out a way to add my Custom Server ID and Keys. and other values using the CLI on headless pi

**Reply by thiswillbeyourgithub** [reply]
Mybe by manually editing the config files? Not sure and that could break things...
